SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.cxx

```diff
@@ -139,6 +139,8 @@ TArray<Worker_ComponentId>{ // Actor components
                             ACTOR_TAG_COMPONENT_ID,
 
                             ACTOR_OWNERSHIP_COMPONENT_ID,
+
+                            FLESHOUT_QUERY_TAG_COMPONENT_ID,
```

Comment:
Why do clients need to see it ?

Reply:
The reasoning is that the ActorSystem is active during this time, and should the client get a skeleton entity for whatever reason, it'd try to create an actor for it. Adding this tag to its interest allows us to mitigate this sans https://github.com/spatialos/UnrealGDK/pull/3160#discussion_r644880654

Reply:
Make sense.